internal/controlapi/server_test.go

```
@@ -593,6 +594,79 @@ func setupHandler(t *testing.T) (*Server, *memoryStore, *mutableChain, *controlp
+	defer journal.Close()
+	readiness := &readinessStub{}
+	metricsKey := []byte(strings.Repeat("m", 32))
+	server, err := NewServer(ServerConfig{Store: store, Lifecycle: lifecycle, Chain: chain, Readiness: readiness, MetricsKey: metricsKey})
+	if err != nil {
+		t.Fatal(err)
+	}
+	httpServer := httptest.NewServer(server)
+	defer httpServer.Close()
+
+	for path, want := range map[string]int{"/livez": http.StatusOK, "/readyz": http.StatusOK, "/health": http.StatusOK} {
+		response, err := httpServer.Client().Get(httpServer.URL + path)
```

golangci-lint (2.12.2)

[error] (*net/http.Client).Get must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)